examples/cryptocurrency/tests/constants/mod.rs

```diff
@@ -0,0 +1,20 @@
+// Copyright 2017 The Exonum Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Shared constants used both in api and transactions logic tests.
+
+/// Alice's wallets name.
+pub const ALICE_NAME: &str = "Alice";
+/// Bob's wallet name.
+pub const BOB_NAME: &str = "Bob";
```

Comment:
Why `constants/mod.rs` instead of just `constants.rs`?

Reply:
https://doc.rust-lang.org/book/second-edition/ch11-03-test-organization.html#submodules-in-integration-tests 🙃

Reply:
Interesting official hack, didn't know about that.